mm2src/crypto/Cargo.toml

```diff
@@ -7,17 +7,24 @@ edition = "2018"
 doctest = false
 
 [dependencies]
+aes = "0.8.3"
+argon2 = { version = "0.5.2", features = ["zeroize"] }
 arrayref = "0.3"
 async-trait = "0.1"
+base64 = "0.11.0"
```

Comment:
Why +4 years old version? We have way too recent versions in our dependency tree already.

Reply:
I guess I used the one imported explicitly in our crates, I will use the latest one from the dependency tree for all crates `v0.21.2`.

Reply:
Done